src/main/kotlin/no/nav/syfo/client/pdl/PdlPersonResponse.kt

```diff
@@ -32,7 +34,15 @@ data class PdlErrorExtension(
 
 data class PdlHentPerson(
     val hentPerson: PdlPerson?
-) : Serializable
+) : Serializable {
+    fun getTilrettelagtKommunikasjon(): TilrettelagtKommunikasjon? =
```

Comment:
La ikke til dette som en extension function ettersom vi tenker at dette er en sentral egenskap rundt hvordan `PdlHentPerson` skal fungere. Jeg tenker på extension functions som utvidelse av egenskaper i spesifikke scope der det er behov. Også om man vil utvide typer/klasser fra 3. parter, for eks. `String`.
Tar gjerne imot andre tanker rundt dette! 🙌

Reply:
*Response/*request/*dto-klasser tenker jeg er hovedsakelig "dumme" klasser som holder data og ikke skal ha så mye oppførsel definert i funksjoner på klassen. For å beregne en property som her så er for så vidt begge deler OK, så lenge det ikke skaper trøbbel for (de)-serialisering.
For domene-klasser tenker jeg vi godt kunne hatt mer funksjoner som en del av klassen og mindre extension, så bra du tar opp diskusjonen 👍

Reply:
Godt poeng rundt dto klasser 🙌 Enig 😊